charts/ceph-csi-rbd/templates/provisioner-psp.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.provisioner.podSecurityPolicy.enabled -}}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ include "ceph-csi-rbd.provisioner.fullname" . }}
+  labels:
+    app: {{ include "ceph-csi-rbd.name" . }}
+    chart: {{ include "ceph-csi-rbd.chart" . }}
+    component: {{ .Values.provisioner.name }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  allowPrivilegeEscalation: true
+  allowedCapabilities:
+    - 'SYS_ADMIN'
+  fsGroup:
+    rule: RunAsAny
+  privileged: true
```

Comment:
both rbd and cephfs pods are  not  privileged, please take care of it

Reply:
I created these PodSecurityPolicies by looking at current Deployment, StatefulSet and DaemonSet manifests.
 
I had to add `privileged: true` to PSP's for provisioner and nodeplugin separately for both charts because there are containers that require privileged access as seen below:
[ceph-csi-rbd/templates/provisioner-deployment.yaml#L136](https://github.com/ceph/ceph-csi/blob/master/charts/ceph-csi-rbd/templates/provisioner-deployment.yaml#L136)
[ceph-csi-rbd/templates/nodeplugin-daemonset.yaml#L106](https://github.com/ceph/ceph-csi/blob/master/charts/ceph-csi-rbd/templates/nodeplugin-daemonset.yaml#L106)
[ceph-csi-cephfs/templates/provisioner-deployment.yaml#L121](https://github.com/ceph/ceph-csi/blob/master/charts/ceph-csi-cephfs/templates/provisioner-deployment.yaml#L121)
[ceph-csi-cephfs/templates/nodeplugin-daemonset.yaml#L107](https://github.com/ceph/ceph-csi/blob/master/charts/ceph-csi-cephfs/templates/nodeplugin-daemonset.yaml#L107)

This goes for all the access I provided in PSP's. I went over all the manifests in charts line by line and gave the least amount of permissions I could possibly give.

Reply:
I thought i already removed privileged from the provisioner pod as part of https://github.com/ceph/ceph-csi/pull/782.  okay let's keep it I have a plan to remove the privileged access from provisioner pod as that is needed, will take care of it, Thanks :+1: